docs/source/man.rst

```diff
@@ -321,7 +321,7 @@ Scans
 
 Use the ``qpc scan`` command to create, run and manage scans.
 
-A scan defines a collection of network information, including the sources to scan and additional options, such as elective products to omit from the scan and the maximum number of parallel system scans. The creation of a scan references information such as sources so that the act of running the scan is repeatable, without a requirement to reenter network information for each scan attempt.
+A scan defines a collection of network or systems management solution information, including the individual sources to scan, and additional options, such as elective products to omit from the scan and the maximum number of parallel system scans. The creation of a scan references information such as sources so that the act of running the scan is repeatable, without a requirement to reenter this information for each scan attempt.
```

Comment:
again this "systems management solution information" just sounds wild to my ear.

Maybe "A scan defines a collection of sources of any type and additional options, such as elective products to omit from the scan and the maximum number of parallel system scans. The creation of a scan references ..."

Reply:
Acknowledged, see other responses.

Reply:
FYI, once I started picking at that paragraph I decided that it needed some extra info. So here's what it evolved into:

A scan is a set of one or more sources of any type plus additional options that refine how the scan runs, such as the products to omit from the scan and the maximum number of parallel system scans. Because a scan can combine sources of different types, you can include network and systems management solution (such as Satellite and vCenter Server) sources in a single scan. When you configure a scan to include multiple sources of different types, for example, a network source and a satellite source, the same part of your infrastructure might be scanned more than once. The results for this type of scan could show duplicate information in the reported results. However, you have the option to view the unprocessed, detailed report that would show these duplicate results, or a summary report where the deduplication and merge processing removes the duplicate results.

Reply:
> the same part of your infrastructure might be scanned more than once. ~The results for this type of scan could show duplicate information in the reported results. However, y~ You have the option to view the unprocessed, detailed report that would show results from each source, where a singe entity may appear more than once, or a summary report where the deduplication and merge processing removes the duplicate results.

Reply:
arhg! Finally logged my firefox into my work account :D